libcontainer/cgroups/devices_emulator.go

```diff
@@ -0,0 +1,351 @@
+// +build linux
+
+package cgroups
```

Comment:
Maybe we want to create a dedicated package for `libcontainer/cgroups/devicesemulator`

Reply:
I've put it in `libcontainer/cgroups/devices` so we can write `devices.Emulator`.